### PR TITLE
[CF-1039] Caspian cluster

### PR DESCRIPTION
**Pull request by ChandraAddala**
- Upgraded spark to 1.3.1
 - Added stop on sparkContext
 - Created case class to grab preference rows

**DO NOT merge this till we are ready to go to caspian cluster**

### CODE REVIEW

File: tiamat/src/main/scala/com/rackspace/feeds/archives/Archiver.scala [hunk 1]
```diff
@@ -91,19 +91,25 @@ class Archiver( runConfig : RunConfig ) {
...
+      val (writtenSet, errorsWrite) = writer.writeFeedsWithEntries(grouped)
 
-    errorsImp ++ errorsWrite ++ errorsWriteEmpty
+      val errorsWriteEmpty = writer.writeFeedsWithoutEntries(writtenSet, runConfig, spark)
+
+      errorsImp ++ errorsWrite ++ errorsWriteEmpty
+
+    } finally {
+      spark.stop()
+    }
```
Comment (ChandraAddala) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Archiver.scala]: Adding try/finally to add spark.stop is the only change here.

**Comment by ChandraAddala** [comment]
** DO NOT merge this till we are ready to go to caspian cluster. **

**Comment by usnavi** [comment]
@ChandraAddala  - I noticed that SPARK_HOME is set at https://github.com/rackerlabs/cloudfeeds-nabu/blob/master/tiamat/src/main/resources/tiamat.sh#L3

Looks like it was removed on the caspian cluster.  Is this set somewhere else?

**Comment by ChandraAddala** [comment]
@usnavi spark-shell, spark-submit etc. are all in the default PATH in /usr/bin. So I didnt have to set SPARK_HOME to find the spark-submit script.

**Comment by usnavi** [comment]
We should probably remove the SPARK_HOME line in this PR then.

**Comment by shintasmith** [comment]
@ChandraAddala, you will have to rebase this one. Thx

**Comment by usnavi** [comment]
lgtm

**Pull request merged**